client/src/main/java/model/Position.java

```diff
@@ -28,23 +28,23 @@ static Position left(Position position) {
     return new Position(position.getColumn() - 1, position.getRow());
   }
 
-  public Integer getColumn() {
+  private Integer getColumn() {
     return column;
   }
 
-  public Integer getRow() {
+  private Integer getRow() {
     return row;
   }
 
   @Override
-  public boolean equals(Object o) {
-    if (this == o) {
+  public boolean equals(Object obj) {
```

Comment:
Do you hold Position in set?

Reply:
Position is more abstract, every sea-field has position, every boundary, every mast etc, so sometimes we want top compare position for different types of fields.